DiffTests/NestedDiffTests.swift

```diff
@@ -0,0 +1,39 @@
+
+import XCTest
+@testable import Diff
+
+/*
+ * empty to something
+ * something to empty
+ * empty to empty
+ * same
+ * different number of sections
+ */
+class NestedDiffTests: XCTestCase {
+
+    func testDiffOutputs() {
+
+        let expectations: [([[Int]], [[Int]], String)] = [
+            ([], [[1, 2], [1]], "I(0,0)I(1,0)I(0,1)"),
+            ([[1, 2], []], [], "D(0, 0)D(1, 0)"),
```

Comment:
It's a `Diff`, not `Patch`. See [here](https://github.com/wokalski/Diff.swift#diff-vs-patch-sorting) for more info.